src/components/side_panel/chart/gauge_chart_panel/gauge_chart_design_panel.ts

```diff
@@ -102,6 +103,10 @@ export class GaugeChartDesignPanel extends Component<Props, SpreadsheetChildEnv>
     );
   }
 
+  get defaultChartTitleFontSize(): number {
```

Comment:
```suggestion
  get defaultGaugeTitleFontSize(): number {
```
nitpick. Same for scorecards